TA.ArduinoPowerController.DeviceInterface/TA.ArduinoPowerController.DeviceInterface.csproj

```diff
@@ -78,16 +78,19 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="JetBrains.Annotations">
-      <Version>2019.1.3</Version>
+      <Version>2021.1.0-eap01</Version>
```

Comment:
I try not to use EAP (prerelease) versions of packages unless there's a strong justification. Consider using the last stable released version?